CalculatingFunctions.js

```diff
@@ -0,0 +1,24 @@
+const calcBasketValue = (basketValue, fallingObjectValue) => {
+    basketValue = basketValue + fallingObjectValue
+}
+
+const calcScore = (basketValue) => {
+    if (basketValue === 1) {
+        ++score;
+        resetBasketValue;
+    }
+}
+
+const calcLives = (basketValue) => {
+    if (basketValue > 1) {
+        --livesRemaining;
+        resetBasketValue;
+    }
+}
+
+// function called when fallingObject touches basket
+function update(fallingObjectValue, basketValue) {
+    calcBasketValue(basketValue, fallingObjectValue);
+    calcScore(basketValue);
+    calcLives(basketValue);
```

Comment:
`calcLives` seems to be the state mutation that occurs when you miss a falling obj, rather than when you catch one. So I don't think it should be called in here.

The prev 2 calls make sense though!

Reply:
Ignore this. I misunderstood when we calculate losing lives. Your flow makes sense